README.md

```diff
@@ -170,6 +170,37 @@ If you would like to get more than just the `id` from the record, you can use th
   def handle_info({:updated, MyApp.Posts.Comment, %{id: id, post_id: post_id}}, socket) do
 ```
 
+## Watching without a schema
+
+Since ecto supports working with tables withoun needed a schema, you may also want to create EctoWatch watchers without needing to create a schema like so:
+
+```elixir
+  # setup
+  {EctoWatch,
+   repo: MyApp.Repo,
+   pub_sub: MyApp.PubSub,
+   watchers: [
+    {
+      %{
+        table_name: "comments",
+        primary_key: "ID",
+        columns: [:title, :body, :author_id, :post_id],
+        association_columns: [:author_id, :post_id]
+      }, :updated, extra_columns: [:post_id]
+    }
+   ]}
+```
+
+Everything works the same as with a schema, though make sure to specify your association columns if you want to subscribe to an association column.
+
+Supported keys for configuring a table without a schema:
+
+ * `pg_schema_name` (optional, defaults to `public`)
```

Comment:
Should `pg_schema_name` maybe be `prefix` to be in line with Ecto nomenclature?

Reply:
Yeah, good point.  I would even say `schema_prefix` because that's what the module attribute is and you call `__schema__(:prefix)` to reflect upon it 👍